### DISCUSSION TITLE
comment not appearing

### DISCUSSION THREAD

**Discussion by novaknole** [KaTeX/KaTeX#3815] [Q&A]
I'm trying to have a comment after the equation right away on the same line. I do:

x = 2 % here it is

problem is, comment doesn't appear. you can try in the editor here: https://katex.org/

**Comment by edemaine** [comment]
That is how `%` comments are supposed to work. They let you leave notes to yourself without anything appearing in the output.

If you want some text to appear, you can use something like `\quad\text{comment}`.
